superset-frontend/cypress-base/cypress/integration/dashboard/edit_mode.js

```diff
@@ -23,37 +23,22 @@ export default () =>
     beforeEach(() => {
       cy.server();
       cy.login();
-
       cy.visit(WORLD_HEALTH_DASHBOARD);
-      cy.get('#app').then(data => {
-        const bootstrapData = JSON.parse(data[0].dataset.bootstrap);
-        const dashboard = bootstrapData.dashboard_data;
-        const dashboardId = dashboard.id;
-        const boxplotChartId = dashboard.slices.find(
-          slice => slice.form_data.viz_type === 'box_plot',
-        ).slice_id;
-        const formData = `{"slice_id":${boxplotChartId}}`;
-        const boxplotRequest = `/superset/explore_json/?form_data=${formData}&dashboard_id=${dashboardId}`;
-        cy.route('POST', boxplotRequest).as('boxplotRequest');
-      });
-
       cy.get('.dashboard-header')
         .contains('Edit dashboard')
         .click();
     });
 
     it('remove, and add chart flow', () => {
-      // wait box_plot data and find box plot
-      cy.wait('@boxplotRequest');
-      cy.get('.grid-container .box_plot').should('be.exist');
+      // wait for box plot to appear
+      cy.get('.grid-container .box_plot');
```

Comment:
A simple `get` will wait for the element to appear. No need to wait for the additional request.